multivac/fetch.py

```diff
@@ -327,75 +325,77 @@ def download_workflow_run_jobs(workflow_run_id):
         yield WorkflowRunJob(data)
 
 
-if not os.path.isdir(workflow_runs_dir):
-    os.makedirs(workflow_runs_dir)
-if not os.path.isdir(workflow_run_jobs_dir):
-    os.makedirs(workflow_run_jobs_dir)
-
-ignore_in_stop_condition = set()
-
-for run in download_workflow_runs(args.branch, args.since):
-    # Stop condition.
-    #
-    # If there are no stored runs, continue till the end (how much
-    # GitHub allows to download, it is 1000 runs).
-    #
-    # However we don't stop on a first known workflow run.
-    # A restarted workflow run keeps its position in the list
-    # (see [1]). So we re-check known runs and stop only when
-    # reach two weeks old runs (unlikely somebody will restart
-    # them).
-    #
-    # Actually this is not the optimal traverse algorithm, but
-    # it is simple to implement.
-    #
-    # [1]: https://github.community/t/135654
-    is_ignored = run.id in ignore_in_stop_condition
-    run_is_old = startup_time - run.created_at > datetime.timedelta(weeks=2)
-    if not args.nostop and run.is_stored and run_is_old and not is_ignored:
-        info('Found stored workflow run {} older than 2 weeks, stopping...',
-             run.id)
-        break
-
-    # Skip incomplete runs. We'll look at them next time.
-    if run.status != 'completed':
-        reason = 'incomplete'
-        info('Skip workflow run {}: {}', run.id, reason)
-        continue
-
-    # Skip already processed runs if there were no restarts.
-    if run.is_stored:
-        run_past_info = WorkflowRun(filepath=run.meta_path)
-        if run.updated_at == run_past_info.updated_at:
-            info(("Workflow run {} was not changed ({}), don't download " +
-                 "jobs again"), run.id, run.updated_at)
+if __name__ == "__main__":
```

Comment:
I use single quotes across the code (if the opposite is not required due to the content of the string). It would be good to keep consistency.

Reply:
Thank you! I fixed all the comments